modules/Logger.js

```diff
@@ -0,0 +1,169 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.Logger = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var Logger = ServiceCell.extend({ 
```

Comment:
I wonder if Logger is the right name for this service. Logger typically concerns logging info/debug/warn/error. This sounds like a session tracker or an event tracker